DAO/project/contracts/DAO-contract/tests/functions/info/governance_asset_id.rs

```diff
@@ -0,0 +1,27 @@
+use crate::utils::{interface::info::governance_asset_id, setup::setup};
+
+mod success {
+    use super::*;
+    use crate::utils::interface::core::constructor;
+
+    #[tokio::test]
+    pub async fn user_can_get_governance_asset_id() {
```

Comment:
Whats with these new tests?

Reply:
Nothing, it is just renamed to remove the word “token”